sdks/java/core/src/test/java/org/apache/beam/sdk/io/DefaultFilenamePolicyTest.java

```diff
@@ -28,6 +31,12 @@
 @RunWith(JUnit4.class)
 public class DefaultFilenamePolicyTest {
 
+  @Before
+  public void setup() {
+    // TODO: Fix failing on windows, https://issues.apache.org/jira/browse/BEAM-10683.
```

Comment:
Can you create a Jira issue per failing test suite as subtasks of BEAM-10683 please? Having a list of failing test suites will be easier for tracking and hopefully encourage people to chip away at it instead of one large issue for them all.

Reply:
Sure. I created 26 subtasks for https://issues.apache.org/jira/browse/BEAM-10683
Every failing test or testing class have link to relevant JIRA issue now.